docroot/modules/custom/sitenow_migrate/sitenow_migrate.install

```diff
@@ -41,3 +42,47 @@ function sitenow_migrate_install() {
     }
   }
 }
+
+/**
+ * Implements hook_install().
+ */
+function sitenow_migrate_migration_install($sitename) {
```

Comment:
Docblock says `hook_install` but this looks like `hook_migration_install` - is this intended?

Reply:
This is a helper function. There is no `hook_migration_install` function.